dashboard/client/src/service/placementGroup.ts

```diff
@@ -0,0 +1,14 @@
+import { PlacementGroup } from "../type/placementGroup";
+import { get } from "./requestHandlers";
+
+export const getPlacementGroup = () => {
+  return get<{
+    result: boolean;
+    message: string;
+    data: {
+      [result: string]: {
+        [result: string]: Array<PlacementGroup>;
+      };
+    };
+  }>("api/v0/placement_groups?detail=1");
```

Comment:
are you sure `&timeout=x` isn't needed here? I tried manually hitting the endpoint and got:
```json
{
  "result": false,
  "msg": "Traceback (most recent call last):\n  File \"/home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/optional_utils.py\", line 95, in _handler_route\n    return await handler(bind_info.instance, req)\n  File \"/home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/modules/state/state_head.py\", line 109, in async_wrapper\n    ret = await func(self, *args, **kwargs)\n  File \"/home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/modules/state/state_head.py\", line 294, in list_placement_groups\n    return await self._handle_list_api(self._state_api.list_placement_groups, req)\n  File \"/home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/modules/state/state_head.py\", line 253, in _handle_list_api\n    result = await list_api_fn(option=self._options_from_req(req))\n  File \"/home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/modules/state/state_head.py\", line 178, in _options_from_req\n    timeout = int(req.query.get(\"timeout\"))\nTypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'\n",
  "data": {}
}
```
guess it's required, but we should consider setting default timeout vals. btw this was on the nightly wheel

Reply:
Do we have in docs an example /api/v0/placement_groups response btw?

Reply:
We don't have a doc. It is something we should do when we migrate dashboard to fast API

Reply:
(along with auto generating api schema)